### PR TITLE
Test for non root user capabilities

### PR DESCRIPTION
**Pull request by mrunalp**
This depends on #1539 which should be merged first.

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
/test all

**Comment by mrunalp** [comment]
Updated.

File: test/ctr.bats [hunk 1]
```diff
@@ -1105,3 +1105,36 @@ function teardown() {
...
+	echo "$newconfig" > "$TESTDIR"/container_user.json
+
+	run crictl create "$pod_id" "$TESTDIR"/container_user.json "$TESTDATA"/sandbox_config.json
+	echo "$output"
+	[ "$status" -eq 0 ]
+	ctr_id="$output"
+	run crictl start "$ctr_id"
+	[ "$status" -eq 0 ]
+
+	run crictl exec --sync "$ctr_id" cat /proc/1/status
```
Comment (rhatdan) [inline test/ctr.bats]: Why not just do
```
run crictl exec --sync "$ctr_id" grep "CapEff:\s0000000000000000" /proc/self/status
```

File: test/ctr.bats [hunk 2]
```diff
@@ -1105,3 +1105,36 @@ function teardown() {
...
+	echo "$output"
+	[ "$status" -eq 0 ]
+	grep "CapEff:\s0000000000000000" $output
+
+	run crictl stopp "$pod_id"
```
Comment (rhatdan) [inline test/ctr.bats]: You should have a 
crictl stop $ctr_id
crictl rm $ctr_id

**Comment by runcom** [comment]
LGTM (modulo Dan's comments!)

File: test/ctr.bats (hunk 1 above)
Comment (mrunalp) [reply test/ctr.bats]: Yeah, I did that.

File: test/ctr.bats (hunk 2 above)
Comment (mrunalp) [reply test/ctr.bats]: That isn't required as stopping/removing the pod takes care of it.

**Comment by mrunalp** [comment]
This is green!

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**